cmd/start.go

```diff
@@ -247,3 +251,34 @@ func initializeDiagnostics(
 	diagnostics.RegisterConnectedPeersSource(registry, netProvider)
 	diagnostics.RegisterClientInfoSource(registry, netProvider)
 }
+
+func initializeBalanceMonitoring(
+	ctx context.Context,
+	chainHandle eth.Handle,
+	config *config.Config,
+	ethereumAddress string,
+) {
+	balanceMonitor, err := chainHandle.BalanceMonitor()
+	if err != nil {
+		logger.Errorf("error obtaining balance monitor handle [%v]", err)
```

Comment:
Don't we want to return just like for stake monitoring? If there's a problem here, then I think it will blow up somewhere down the road, ex. line 271
```
return fmt.Errorf("error obtaining balance monitor handle: [%v]", err)
```

Reply:
Yep, good catch! I missed a `return` here.